manifests/proxydhcp.pp

```diff
@@ -49,6 +49,12 @@
     failover       => $failover,
   }
 
+  exec { 'setfacl_dhcp':
+    command => 'setfacl -R -m u:theforeman:rwx /etc/dhcp /var/lib/dhcpd',
+    path => '/usr/bin',
+    onlyif => 'test -f /var/lib/dhcp && test -f /etc/dhcp',
```

Comment:
That helps some, but not in understanding what this needs to do to handle if they are disabled. Consider me in over my head :)